src/poprox_recommender/evaluation/evaluate.py

```diff
@@ -27,14 +28,21 @@
 from lenskit.metrics import topn
 from tqdm import tqdm
 
-from poprox_concepts.api.recommendations import RecommendationRequest
 from poprox_concepts.domain import Article, ArticleSet
+from poprox_recommender.config import available_cpu_parallelism
 from poprox_recommender.data.mind import MindData
 from poprox_recommender.evaluation.metrics import rank_biased_overlap
 from poprox_recommender.logging_config import setup_logging
 from poprox_recommender.paths import project_root
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger("poprox_recommender.evaluation.evaluate")
```

Comment:
executable scripts need this, because __name__ will be __main__ when they are run.